### DISCUSSION TITLE
Proxy set through uc.ChromeOptions() is not used

### DISCUSSION THREAD

**Discussion by Nygosaki** [ultrafunkamsterdam/undetected-chromedriver#1779] [Q&A]
After setting the proxy and navigating to https://whatismyipaddress.com, the ip address does not change to that of the proxies.
This is my code,
```python
        self.chromeOptions = uc.ChromeOptions()
        self.chromeOptions.add_argument('--ignore-ssl-errors=yes')
        self.chromeOptions.add_argument('--ignore-certificate-errors')
        self.chromeOptions.add_argument("--headless")
        self.chromeOptions.add_argument('--log-level=3')
        self.chromeOptions.add_argument('--disable-blink-features=AutomationControlled')            
        PROXY = proxyIP + ":" + proxyPort
        self.chromeOptions.add_argument('--proxy-server=socks5://%s' % PROXY)
        driver = uc.Chrome(Options=self.chromeOptions, version_main=int(getChromeVer.get_chrome_version().split(".")[0]))
        driver.get("https://whatismyipaddress.com")
```

**Comment by KebabLord** [comment]
can you verify that starting chrome with proxy parameter works?
`google-chrome --proxy-server=socks5://yourip:yourport`

**Comment by sethmenghi** [comment]
Is it because Options is set with Options (capital O?) not options?

**Comment by Nygosaki** [comment]
In the end, I decided to switch to nodriver instead. I would recommend anybody using this library to do so as well, as I think UC isn't getting worked on anymore due to nodriver

**Reply by raunaqss** [reply]
Did you need to make changes to get it working with nodriver?

**Reply by cu0302pid** [reply]
> In the end, I decided to switch to nodriver instead. I would recommend anybody using this library to do so as well, as I think UC isn't getting worked on anymore due to nodriver

for me also not working connecting proxy through nodriver uc
